### PR TITLE
Database setup

### PR DESCRIPTION
**Pull request by Julian-Jurai**
You must first create a database with the name specified in the `config.json` and enter you credentials for your postgres user. There must be a better way to handle this but oh well for now..

I added a barebones format of how the controllers and models interact with the DB.

to test it out go to: `localhost:8000//users/create` and use post man to create a post request with a `first_name`.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -24,30 +24,30 @@ app.use('/', indexRouter);
...
+// );
+
+// app.get('/auth/facebook', passport.authenticate('facebook'));
+
+// app.get('/auth/facebook/callback',
+//   passport.authenticate('facebook', { failureRedirect: '/login' }),
+//   (req, res) => {
+//     res.redirect('/');
+//   }
+// )
```
Comment (Julian-Jurai) [inline server/app.js]: This was giving some errors so I commented it out for the while

Comment (sksea) [reply server/app.js]: The error's caused by not having the facebook app ID and secret set in environment. Let me send you those if you don't have already.

File: server/controllers/users.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const User = require('../db/models').User;
+
+module.exports = {
+  create(req, res) {
+		console.log('in users controller')
```
Comment (sksea) [inline server/controllers/users.js]: We should stick to spaces instead of tabs.

File: server/db/migrations/20180422175455-create-user.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+'use strict';
```
Comment (sksea) [inline server/db/migrations/20180422175455-create-user.js]: Should we add `use strict` to all our js files?

File: README.md [hunk 4]
```diff
@@ -9,6 +9,14 @@
 
 ### Database
 
+### Creating a model and migration
```
Comment (sksea) [inline README.md]: Use h4's or bold to 'nest' under Database header.

File: server/app.js (hunk 1 above)
Comment (sksea) [reply server/app.js]: The errors are probably caused by not setting the FB app id and secret in environment. We should keep this in and let me send you the envvars.

File: server/routes/users.js [hunk 5]
```diff
@@ -1,9 +1,15 @@
 var express = require('express');
 var router = express.Router();
+var usersController = require('../controllers').users;
 
 /* GET users listing. */
 router.get('/', function(req, res, next) {
   res.json({users: ['albert', 'julian', 'andy']})
 });
+// router.get('/', usersController.getUsers)
```
Comment (sksea) [inline server/routes/users.js]: We should prefer not having commented out code because they're easy to forget about.

File: server/app.js (hunk 1 above)
Comment (sksea) [reply server/app.js]: Errors caused by not having set FB app ID and secret. Let's keep this in and I'll send you the env vars.

**Pull request merged**